hm_08/hm_08/CheckIn.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace hm_08
+{
+    public class CheckIn 
+    {
+        public const string read = "Для продолжения нажмите любую клавишу";       
+        public  string Usname { get; set; }
+        public void Reg(CheckIn check)
+        {
+            string usname = check.Usname; 
+            Console.WriteLine($" {usname},вы проходили онлайн регистрацию?  \n Введите Y или N");
+            string ans = Console.ReadLine();
+            while (ans != "Y" && ans != "N")
```

Comment:
As you remember, I mentioned that if we need to compare strings the best practice to use **Equals** method instead of **!=**. **==** operators.